src/sparseml/pytorch/image_classification/export.py

```diff
@@ -368,7 +386,13 @@ def main(
 
     if recipe is not None:
         ScheduledModifierManager.from_yaml(recipe).apply_structure(model)
+
+    if checkpoint_path:
```

Comment:
what is the reason for this change? could this potentially fail before?

Reply:
Before, the code wasn't loading in checkpoint weights if the recipe was not provided which was a bug; this fixes that